gateway-developer-plugin/src/main/java/com/ca/apim/gateway/cagatewayconfig/BuildDeploymentBundleTask.java

```diff
@@ -66,6 +76,9 @@ public void perform() {
         BundleFileBuilder bundleFileBuilder = InjectionRegistry.getInjector().getInstance(BundleFileBuilder.class);
         final ProjectInfo projectInfo = new ProjectInfo(getProject().getName(), getProject().getGroup().toString(),
                 getProject().getVersion().toString(), null);
+        if(targetFolder.isPresent()){
+            projectInfo.withTargetFolder(targetFolder.get());
```

Comment:
You can convert this `withTargetFolder()` to setter method. It is doing that only.